src/ccs/confluent.sh

```diff
@@ -993,6 +993,17 @@ list_command() {
     fi
 }
 
+# CLI-84: If using Mac, version must be >= 10.13
+validate_os_version() {
+    local OS=$(uname -s)
+    if [[ "${OS}" = "Darwin" ]]; then
+        local mac_version=$(sw_vers -productVersion)
+        if [[ $(awk 'BEGIN {print ("'${mac_version}'" < "'10.16'")}') )) ]]; then
+            die "WARNING: MacOS version >= 10.13 is required for all Confluent features to work."
```

Comment:
It should probably explicitly say " Confluent CLI will exit." (like the java message).  Otherwise users may think CP *was* started.

Reply:
I think adding `Error` covers this case.

Reply:
Ok, just left as ERROR for now, if you want something else just ping me